aiohttp/multipart.py

```diff
@@ -907,16 +901,14 @@ def write(self, chunk):
                     return
 
         if self._encoding == 'base64':
-            self._encoding_buffer.extend(chunk)
-
-            if self._encoding_buffer:
-                buffer = self._encoding_buffer
-                div, mod = divmod(len(buffer), 3)
-                enc_chunk, self._encoding_buffer = (
-                    buffer[:div * 3], buffer[div * 3:])
-                if enc_chunk:
-                    enc_chunk = base64.b64encode(enc_chunk)
-                    yield from self._writer.write(enc_chunk)
+            if chunk:
+                enc_chunk = base64.encodebytes(chunk)
```

Comment:
> Are you serious?

What's your point?

This is a work in progress. As you can see in my [previous comment](https://github.com/aio-libs/aiohttp/pull/2128#issuecomment-321599180) I'm already aware of this problem.

The previous code that you show indeed fixes the problem of the multiple read chunks, but doesn't adress the long line problem that I reported initially. I will try to solve these two constraints soon (I hope).